src/basho_bench_stats.erl

```diff
@@ -67,6 +67,8 @@ op_complete(Op, {ok, Units}, ElapsedUs) ->
             gen_server:cast({global, ?MODULE}, {Op, {ok, Units}, ElapsedUs});
         false ->
             folsom_metrics:notify_existing_metric({latencies, Op}, ElapsedUs, histogram),
+            folsom_metrics:notify_existing_metric({overall_latencies, Op}, ElapsedUs, histogram),
+            folsom_metrics:notify_existing_metric({overall_units, Op}, {inc, Units}, counter),
```

Comment:
So yea, we could sum over the `n` from the other data set, but the other data set keeps getting cleared after every window, meaning we'd have to keep track of it somewhere else. Seemed cleaner to just run a separate counter